pypuf/simulation.py

```diff
@@ -76,18 +76,28 @@ def normal_weights(n, k, mu=0, sigma=1):
         """
         return random.normal(loc=mu, scale=sigma, size=(k, n))
 
-    def __init__(self, weight_array, transform, combiner):
+    def __init__(self, weight_array, transform, combiner, bias=False):
+        """
+        Constructs an LTFArray based on given weight_array and
+        combiner function with appropriate transformation of challenges.
+        The bias is committed through the (n+1)th value in weight_array.
+        So the parameter bias only states if the given weight_array has
+        n+1 values (or not) while the challenges still has length n.
+        """
         (self.k, self.n) = shape(weight_array)
         self.weight_array = weight_array
         self.transform = transform
         self.combiner = combiner
+        self.bias = bias
 
     def eval(self, inputs):
         """
-        evaluates a given list of challenges
+        evaluates a given list of challenges regarding bias
         :param x: list of challenges
         :return: list of responses
         """
+        if self.bias:
+            inputs = tools.iter_append_last(inputs, 0)
```

Comment:
Shoudn't that be a `1` instead of a `0`? I believe that the `0` will not change the dot product that is evaluated in `self.val`